lib/Doctrine/Migration.php

```diff
@@ -307,15 +307,13 @@ public function getNextMigrationClassVersion()
      *
      * @param  integer $to       Version to migrate to
      * @param  boolean $dryRun   Whether or not to run the migrate process as a dry run
-     * @return integer $to       Version number migrated to
+     * @return bool              True if the migration succeeded, false otherwise
```

Comment:
As I read the code.

It never returns true. Otherwise it returns the version migrated to.

Also the behaviour depends on the dryRun flag.

Reply:
oh, right. it returns either the $to (integer) or 'false'